backend/kesaseteli/common/utils.py

```diff
@@ -16,21 +16,38 @@ def has_permission(self, request, view):
         return False
 
 
+def has_whitespace(value):
+    value_without_whitespace = "".join(value.split())
+    return value != value_without_whitespace
+
+
+def is_uppercase(value):
+    return value == value.upper()
+
+
 def validate_finnish_social_security_number(value):
     """
-    Raise a ValidationError if the given value is not a Finnish social security number.
+    Raise a ValidationError if the given value is not an uppercase Finnish social
+    security number with no whitespace.
     """
-    if not is_valid_finnish_social_security_number(value):
+    if (
+        not is_valid_finnish_social_security_number(value)
+        or has_whitespace(value)
+        or not is_uppercase(value)
```

Comment:
You can just use `not value.isupper()`

Reply:
Thanks for the tip! Taking into account.

Reply:
Actually it's not the same. "123".isupper() == False which is weird. I'm changing is back to the former way...

Reply:
I changed this back and added documentation to is_uppercase function that it's not same as string.isupper